src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/EventType.java

```diff
@@ -25,6 +25,7 @@ public enum EventType {
     CLOSE("FR_close"),
     MANAGE_CASE_DOCUMENTS("FR_manageCaseDocuments"),
     UPLOAD_APPROVED_ORDER("FR_uploadApprovedOrder"),
+    FR_NEW_PAPER_CASE_EVENT("FR_newPaperCase"),
```

Comment:
This change - where this has been used. Seems not needed. please check.

Reply:
Yes, Rishi forgot to remove this enum, thanks